support/cas-server-support-memcached-ticket-registry/src/main/java/org/apereo/cas/ticket/registry/MemCacheTicketRegistry.java

```diff
@@ -120,7 +120,7 @@ public Ticket getTicket(final String ticketIdToGet) {
     @Override
     public Collection<Ticket> getTickets() {
         LOGGER.debug("getTickets() isn't supported. Returning empty list");
-        return new ArrayList<>();
+        return Collections.emptyList();
```

Comment:
In fact, I'd even go so far as to say, this sort of thing probably needs to be enforced by checkstyle :)